packages/flutterfire_ui/lib/src/auth/oauth/providers/apple_provider.dart

```diff
@@ -94,6 +94,8 @@ class AppleProviderConfiguration extends OAuthProviderConfiguration {
   @override
   bool isSupportedPlatform(TargetPlatform platform) {
     return !kIsWeb &&
-        (platform == TargetPlatform.iOS || platform == TargetPlatform.macOS);
+        (platform == TargetPlatform.iOS ||
+            platform == TargetPlatform.macOS ||
+            kIsWeb);
```

Comment:
even though [sign_in_with_apple](https://pub.dev/packages/sign_in_with_apple) readme mentions it works on the web, there's also a section that mentions a custom server (glitch.com). I wasn't able to sign in with apple on web (see this issue)[https://github.com/aboutyou/dart_packages/issues/269], but this is something that will be supported eventually

Reply:
Thank you for review, I revered apple provider.